Gaming The Syllabus/UserSettings/EditorUserSettings.asset

```diff
@@ -6,34 +6,34 @@ EditorUserSettings:
   serializedVersion: 4
```

Comment:
We really need to sort out our gitignore to make sure this doesn't keep getting committed. Will look into it when I have time. We might need to force git to clear cached files and see if that helps. Will keep everyone updated on what I find out when I look into it